### PR TITLE
DOC-2234: Add workflow files to test building antora 5 docs on staging

### PR DESCRIPTION
**Pull request by MitchC1999**
Ticket: DOC-2234

Changes:
* Added staging and release workflow files
* Workflow should build 5 docs and sync to staging/release
* Should not delete existing staging docs sites.

Pre-checks:
- [x] Branch prefixed with `feature/5/` or `hotfix/5`
- [x] Files has been included where required (if applicable)

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by kemister85 — APPROVED** [review]
LGTM

File: antora-playbook-dev.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    branches: HEAD
+    start_path: ./
+urls:
+  html_extension_style: indexify
+ui:
+  bundle:
+    url: https://tiny-cloud-docs-antora-themes-staging.s3.amazonaws.com/ui-bundle.zip
+asciidoc:
+  attributes:
+    tinymce_live_demo_url: https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/6-dev/tinymce.min.js
```
Comment (ltrouton) [inline antora-playbook-dev.yml:17]: Just wanted to check if this should be pointing to 5-dev instead?

**Review by ltrouton — APPROVED** [review]

File: antora-playbook-dev.yml (hunk 1 above)
Comment (MitchC1999) [reply antora-playbook-dev.yml:17]: Thanks for catching that, and for reminding me about checking the demos during the content migration.

**Review by ShiridiGandham — APPROVED** [review]
LGTM

**Pull request merged**